CMakeLists.txt

```diff
@@ -134,3 +134,14 @@ target_link_libraries(wasp_unittests
   gtest_main
   gtest
 )
+
+add_executable(wasp_sqlite
```

Comment:
Why not a wasp.cc subtool?

Reply:
I didn't want to include sqlite for just the one tool. But yeah, with a different structure maybe it matters less (`git` style w/ separate executables, or using shared objects)

Reply:
Mm. To clarify, either way makes sense to me, this just seemed inconsistent and I wasn't sure what the reason behind it was. Not including sqlite for all the things seems like a pretty good reason.